# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/XRWebGLLayer.cpp
# Commit: bbfe23c61add
# Full Hash: bbfe23c61add07377daf80696fd6996c0463f5b6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 8fcfac450ecc -r bbfe23c61add dom/canvas/XRWebGLLayer.cpp
--- a/dom/canvas/XRWebGLLayer.cpp	Mon Jun 08 20:13:01 2020 +0000
+++ b/dom/canvas/XRWebGLLayer.cpp	Mon Jun 08 20:34:15 2020 +0000
@@ -261,7 +261,6 @@
 
 void XRWebGLLayer::SessionEnded() {
   DeleteFramebuffer();
-  mWebGL->ClearVRFrame();
 }
 
 }  // namespace dom